Sources/BraintreeDropIn/Public/BraintreeDropIn/BTDropInResult.h

```diff
@@ -54,7 +54,7 @@ typedef NS_ENUM(NSInteger, BTDropInErrorType) {
 @property (nonatomic, strong, nullable) NSString *deviceData;
 
 /**
- * Fetch a `BTDropInResult` with the customer's most recently vaulted payment method.
+ * Fetch a `BTDropInResult` with the customer's vaulted payment methods. The order is not guaranteed.
```

Comment:
Ah you're right, good catch, we fetch the entire array but only return the first object in the array, which makes this docs string even more confusing since we cannot actually regularly determine what payment method that will be. I'll update this string.